scripts/populate_institutions.py

```diff
@@ -71,6 +72,7 @@ def main(env):
                 'logout_url': SHIBBOLETH_SP_LOGOUT.format(encode_uri_component('https://osf.io/goodbye')),
                 'domains': [],
                 'email_domains': [],
+                'delegation_protocol': 'SAML_SHIBBOLETH',
```

Comment:
The protocol is `SAML` and the implementation we use is `SHIBBOLETH`.
I reserve `SAML` for using `SAML` through `pac4j` and use `SAML_SHIBBOLETH` for our own Shibboleth Server.